src/connectedk8s/azext_connectedk8s/_params.py

```diff
@@ -20,6 +20,17 @@
 
 def load_arguments(self, _):
 
+    with self.argument_context('connectedk8s troubleshoot') as c:
+        c.argument('tags', tags_type)
+        c.argument('location', arg_type=get_location_type(self.cli_ctx), validator=get_default_location_from_resource_group)
```

Comment:
we don't take tags or location params as input for troubleshoot command. Can remove